core/src/Spirit/Configurations.cpp

```diff
@@ -13,7 +13,9 @@
 #endif
 
 
-std::function< bool(const Vector3&, const Vector3&) > get_filter(Vector3 position, const float r_cut_rectangular[3], float r_cut_cylindrical, float r_cut_spherical, bool inverted)
+std::function<bool(const Vector3&, const Vector3&)> 
+get_filter( Vector3 position, const float r_cut_rectangular[3], float r_cut_cylindrical, 
```

Comment:
The function `get_filter()` and `filter_to_string()` are not directly exposed to the user through the API. Instead they are called by API functions. We do not have to guard for `Exceptions` since those will be thrown to the API functions which are wrapped around `try( ... ) catch( ... )`.